scripts/common/rook.sh

```diff
@@ -181,9 +181,20 @@ function current_rook_version() {
         | sed 's/v\([^-]*\).*/\1/'
 }
 
-# checks if rook should be upgraded before upgrading k8s. If it should, reports that as an addon, and starts the upgrade process.
+# checks if rook should be upgraded before upgrading k8s. If it should, and the user confirms this, reports that as an addon, and starts the upgrade process.
 function maybe_report_upgrade_rook_10_to_14() {
     if should_upgrade_rook_10_to_14; then
+        printf "Upgrading Rook will take some time and will place additional load on your server.\n"
+        if ! [ "$ROOK_BLOCK_STORAGE_ENABLED" = "1" ]; then # this check should probably be moved to golang (as a 'are there actually disks' check instead of a 'does the spec require disks' check
+            printf "In order to complete this migration, you will need to attach a blank disk to each node in the cluster for Rook to use.\n"
+        fi
+        printf "Would you like to continue? "
+
+        if ! confirmN; then
+            printf "Not upgrading Rook\n"
+            return 0
+        fi
+
```

Comment:
```suggestion
        echo "Upgrading Rook will take some time and will place additional load on your server."
        if [ "$ROOK_BLOCK_STORAGE_ENABLED" != "1" ]; then # this check should probably be moved to golang (as a 'are there actually disks' check instead of a 'does the spec require disks' check
            echo "In order to complete this migration, you will need to attach a blank disk to each node in the cluster for Rook to use."
        fi
        printf "Would you like to continue? "

        if ! confirmN; then
            echo "Not upgrading Rook"
            return 0
        fi
```